### PR TITLE
Magic number cleanup part 5

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**

### CODE REVIEW

File: packages/app/src/components/pie-chart.tsx [hunk 1]
```diff
@@ -251,26 +211,9 @@ export function PieChart<T>({
...
-                  />
-                  <Markdown
-                    content={replaceVariablesInText(
-                      item.label,
-                      data as any,
-                      formatters
-                    )}
-                  />
+                <Box as="li" key={`${item.color}-${index}`} display="flex" alignItems="center" spacingHorizontal={2}>
+                  <Box width={'12px'} height={'12px'} backgroundColor={item.color} borderRadius="50%" />
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/components/pie-chart.tsx:215]: Nitpicky comment and non-blocking, but width="12px" would also be sufficient and might be easier to interpret as a static value (the { ... } can imply that this is actually a variable property). The same applies for the height example in this file.

File: packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ export const TimelineTooltipContent = ({ config, hasMultipleEvents, onNext, onPr
       </Box>
 
       {isTouch && (
-        <Box borderTop={`1px solid ${colors.gray2}`} display="flex" justifyContent="center" mx="-27px" pt={space[3]} textVariant="label1">
+        <Box borderTop={`1px solid ${colors.gray2}`} display="flex" justifyContent="center" mx="-27px" paddingTop={space[3]} textVariant="label1">
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx:94]: You could change `mx` to `marginX`

File: packages/app/src/components/pie-chart.tsx (hunk 1 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/pie-chart.tsx:215]: adjusted accordingly, please resolve.

File: packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx:94]: adjusted accordingly, please resolve.

Comment (VWSCoronaDashboard21) [reply packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx:94]: I notice `mx` still on line 94?

**Review by VWSCoronaDashboard26 — APPROVED** [review]
Pre-approving as all my earlier remarks have been resolved.

File: packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx (hunk 2 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/severity-indicator-tile/components/timeline/components/tooltip-content.tsx]: Sorry, missed one

**Pull request merged**